server/metrics/metrics.go

```diff
@@ -291,6 +298,24 @@ func NewMetrics(info InstanceInfo) Metrics {
 	}, []string{"action"})
 	m.registry.MustRegister(m.subscriptionsTotal)
 
+	m.messageHooksEvents = prometheus.NewCounterVec(prometheus.CounterOpts{
+		Namespace:   MetricsNamespace,
+		Subsystem:   MetricsSubsystemHooks,
+		Name:        "events_total",
+		Help:        "The total number of message hooks events (message creation and update, reactions creation and update).",
```

Comment:
nit: propose letting the labels expand on the help here
```suggestion
		Help:        "The total number of hook events related to message processing.",
```